pubspec.yaml

```diff
@@ -1,96 +1,66 @@
 name: characters
-version: 1.1.0-nullsafety
+version: 1.1.0-nullsafety.1
 description: String replacement with operations that are Unicode/grapheme cluster aware.
 homepage: https://www.github.com/dart-lang/characters
 
 environment:
-  # This must remain a tight constraint (only allow dev versions) until nnbd is
-  # stable.
-  sdk: '>=2.9.0-18.0 <2.9.0'
+  # This must remain a tight upper bound constraint until nnbd is stable
+  sdk: '>=2.9.0-18.0 <2.10.0'
```

Comment:
I don't think this will work on a 2.10 dev sdk, since min sdk will be below the "current" language version.

Reply:
Ya, I forgot that we allow you to get a dev sdk with a constraint like this (do we still? not actually sure but we used to).

We might instead want to do like `<2.9.10`? So we don't have to republish for patch releases.

Reply:
going with <=2.9.10 after the meeting today